hail/src/main/scala/is/hail/expr/ir/StringTableReader.scala

```diff
@@ -44,52 +123,32 @@ class StringTableReader(
 
   override def pathsUsed: Seq[String] = params.files
 
-  override def lower(ctx: ExecuteContext, requestedType: TableType): TableStage =
-    executeGeneric(ctx).toTableStage(ctx, requestedType)
+  override def lower(ctx: ExecuteContext, requestedType: TableType): TableStage = {
+    val fs = ctx.fs
+    println(requestedType.rowType)
+    val lines = GenericLines.read(fs, fileStatuses, None, None, params.minPartitions, false, true)
+    TableStage(globals = MakeStruct(FastSeq()),
+      partitioner = RVDPartitioner.unkeyed(lines.nPartitions),
+      dependency = TableStageDependency.none,
+      //TODO figure out if stream requires memory management per element
+      contexts = ToStream(Literal.coerce(TArray(lines.contextType), lines.contexts), true),
```

Comment:
I thought about this, and it's unnecessary. So you can take away the `true` at the end of line 134 and delete the TODO. We need that to be true when the streams represent rows of a table within a partition. This stream is just some helper information to make the partitions.